TESTING.md

```diff
@@ -38,14 +42,19 @@ You can test against a temporary local Datastore by following these steps:
 You can test against a remote Datastore emulator as well.  To do this, set the `DatastoreOptions` project endpoint to the hostname of the remote machine, like the example below.
 
   ```java
-  DatastoreOptions options = LocalDatastoreHelper.options()
-      .toBuilder()
+  DatastoreOptions options = DatastoreOptions.builder()
+      .projectId("my-project-id") // must match project ID specified on remote machine
       .host("http://<hostname of machine>:<port>")
+      .authCredentials(AuthCredentials.noAuth())
       .build();
   Datastore localDatastore = options.service();
   ```
 
-Note that the remote Datastore emulator must be running before your tests are run.
+Note that the remote Datastore emulator must be running before running the code above. We recommend that you start the emulator on the remote machine using the [Google Cloud SDK](https://cloud.google.com/sdk/gcloud/reference/beta/emulators/datastore/) from command line, as shown below:
+
+```
+gcloud beta emulators datastore start
```

Comment:
We should probably show how to set the port. If the emulator has to be remotely accessible I guess users would want it on a specific port.

```
gcloud beta emulators datastore start --host-port <hostname of machine>:<port>
```